wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -29,6 +29,9 @@ <h1 class="w-header__title">
                         <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
                     {% endif %}
                     {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
+                    {% if header_description %}
+                        <div class="w-header__description">{{ header_description }}</div>
+                    {% endif %}
```

Comment:
A reminder to add the `description` to the variables the template accepts at the top of the docs above.